WordPress/Tests/Theme/DiscouragedIframeUnitTest.inc

```diff
@@ -0,0 +1,5 @@
+<?php
+
```

Comment:
Sorry, I was not very clear. What I meant was `echo '<iframe src="http://example.com" >'`